README.md

```diff
@@ -33,6 +32,7 @@ Here is a list of some third-party implementations in other languages.
 
 |    Language    | Clients | Servers | Repository |
 |----------------|---------|---------|------------|
+| **Go**         |    ✓    |    ✓    | [github.com/twitchtv/twirp](https://github.com/twitchtv/twirp)
```

Comment:
Not sure the implementation in Go counts as an `other language`

Reply:
yeah, lol. What was I thinking?